diff -r -u -N geos-3.4.2 _clean/CMakeLists.txt geos-3.4.2/CMakeLists.txt
--- geos-3.4.2 _clean/CMakeLists.txt	Sun Aug 25 15:10:31 2013
+++ geos-3.4.2/CMakeLists.txt	Thu Jan 22 16:04:32 2015
@@ -122,7 +122,7 @@
   #set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W4")
   string(REGEX REPLACE "/W[0-9]" "/W4" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
 
-  if(MSVC80 OR MSVC90 OR MSVC10 OR MSVC11)
+  if(MSVC80 OR MSVC90 OR MSVC10 OR MSVC11 OR MSVC12)
 
     # Option is to enable the /MP switch for Visual Studio 2005 or later
     option(GEOS_MSVC_ENABLE_MP
diff -r -u -N geos-3.4.2 _clean/include/geos/platform.h.cmake geos-3.4.2/include/geos/platform.h.cmake
--- geos-3.4.2 _clean/include/geos/platform.h.cmake	Sun Aug 25 15:10:31 2013
+++ geos-3.4.2/include/geos/platform.h.cmake	Thu Jan 22 16:04:49 2015
@@ -85,6 +85,7 @@
  */
 #ifdef _MSC_VER
 #define _USE_MATH_DEFINES
+#include <algorithm>
 #include <cmath>
 #include <float.h>
 #else
diff -r -u -N geos-3.4.2 _clean/src/operation/buffer/BufferOp.cpp geos-3.4.2/src/operation/buffer/BufferOp.cpp
--- geos-3.4.2 _clean/src/operation/buffer/BufferOp.cpp	Sun Aug 25 15:10:29 2013
+++ geos-3.4.2/src/operation/buffer/BufferOp.cpp	Thu Jan 22 16:06:58 2015
@@ -21,6 +21,10 @@
 #include <algorithm>
 #include <cmath>
 
+#ifdef _MSC_VER
+#define NOMINMAX
+#endif
+
 #include <geos/profiler.h>
 #include <geos/precision/GeometryPrecisionReducer.h>
 #include <geos/operation/buffer/BufferOp.h>
